lessons/specifics/mnesia.md

```diff
@@ -115,7 +115,7 @@ We define the columns using the atoms `:id`, `:name`, and `:job`. When we execut
 
 ## The Dirty Way
 
-First of all we will look at the dirty way of reading and writing to an Mnesia table. This should generally be avoided as success is not guaranteed, but it should help us learn and become comfortable working with Mnesia. Let's add some entries to our **Person** table.
+First of all we will look at the dirty way of reading and writing to a Mnesia table. This should generally be avoided as success is not guaranteed, but it should help us learn and become comfortable working with Mnesia. Let's add some entries to our **Person** table.
```

Comment:
Thanks, @ybur-yug! 
Yes, 'an' is used before words which sound like they start with a vowel even if their first letter is a consonant. But I think that there is no starting `a` in this case. `Mnesia` and [`amnesia`](https://en.wikipedia.org/wiki/Amnesia) are different words with the opposite meanings. I decided to watch several videos about Mnesia to find out the most commonly used case but it didn't really help that much: it seems like there are many options occurring with equal frequency like `əmnɪʒə`, `mnɪʒə`,  `nɪʒə` and so on. If we pronounce `Mnesia` as `əmnɪʒə`, how would we pronounce `amnesia` then?
Maybe @doomspork can shed some light on it.